src/dbnode/server/server.go

```diff
@@ -411,13 +411,16 @@ func Run(runOpts RunOptions) {
 
 	// Setup query stats tracking.
 	statsOpts := stats.QueryStatsOptions{
-		Lookback: stats.DefaultLookback,
+		MaxDocsLookback:      stats.DefaultLookback,
+		MaxBytesReadLookback: stats.DefaultLookback,
```

Comment:
nit: `Period` I think is more descriptive? Probably not worth refactoring all of these though

Reply:
Hm yeah I think from the perspective of applying the limit `lookback` makes more sense. It just happens to be that the implementation is periodic resetting.